yarn.lock

```diff
@@ -1599,14 +1599,6 @@
     rollup-pluginutils "^2.3.3"
     slash "^2.0.0"
 
-"@ovh-ux/manager-navbar@^0.1.0-alpha.1":
```

Comment:
`yarn install` from `4976486a6b89967090fcecbc55e9d0bcd6bb30da` causes this following change.